Dockerfile.openshift

```diff
@@ -12,6 +12,7 @@ ENV GO111MODULE=on
 RUN go build -mod vendor -o bin/egress-router cmd/egress-router/egress-router.go
 
 FROM registry.ci.openshift.org/ocp/4.16:base-rhel9
+RUN dnf install -y util-linux && dnf clean all
```

Comment:
Instead of adding a new RUN line, this could be added to the one already present in order to avoid adding an additional layer to the image.